hypothesis/h
# Migrate from raven-js to @sentry/browser

Migrate frontend crash reporting from Sentry's old raven-js library to the modern `@sentry/browser` JS library.

**Testing:**

1. Copy the DSN value from https://hypothesis.sentry.io/settings/projects/frontend-h/keys/
2. Set the `SENTRY_DSN_FRONTEND` variable in h's environment to the DSN value
3. Load a page in h, you should see a request to https://o18985.ingest.us.sentry.io/api/146908/envelope/
